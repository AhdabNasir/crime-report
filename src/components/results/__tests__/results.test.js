import React from 'react';
import renderer from 'react-test-renderer';
import Results from '..';

describe('<Results/>', () => {
  it('renders <Results/> component', () => {
    const tree = renderer
      .create(<Results number={10} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
