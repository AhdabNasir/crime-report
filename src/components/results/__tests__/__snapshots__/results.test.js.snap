// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Results/> renders <Results/> component 1`] = `
<p
  className="govuk-body"
>
  Showing
   
  10
   
  results
</p>
`;
